File: python/color.py
#Import required modules
import colorama
from colorama import Fore, Back, Style

#Initialize colorama
colorama.init(autoreset=True)

#Print text using background and font colors
print(Back.RED + Fore.BLUE + "Welcome to LinuxHint")
#Add newline
print()
#Print text using background color
print(Back.GREEN + "I like programming")

File: rust/color.rs
struct Color {
	red: i8,
	blue: i8,
	green: i8
}

fn main(){

	let mut bg = Color{red: 18, blue: 20, green: 25};
	println!("{} {} {}", bg.red, bg.blue, bg.green);
	bg.red = 35;
	println!("{} {} {}", bg.red, bg.blue, bg.green);
}
